Makefile

```diff
@@ -4,7 +4,9 @@ DOCKERHUB_USERNAME=k8loud
 IMAGE_NAME=themis-executor
 # w.x.y.z, one digit value each
 # when tinkering add -<description> suffix
-VER=0.0.3.8
+
+VER=0.0.3.8-scaling
```

Comment:
Why newlines?
Bump it to 0.0.4.0, we'll merge it after https://github.com/k8loud/themis-executor/pull/143 since it has 0.0.3.9 already booked